tests/common/plugins/memory_utilization/memory_utilization_common.json

```diff
@@ -0,0 +1,15 @@
+{
+  "COMMON": [
+    {
+      "name": "monit",
+      "cmd": "sudo monit status",
+      "memory_params": {
+        "memory_usage": {
+          "memory_increase_threshold": 5,
+          "memory_high_threshold": 70
```

Comment:
Just question. Any reason why it is 70 not 80?

Reply:
Thanks for asking, Currently, the SONiC Memory Utilization Threshold is 70% in ICM. So we set 70 here in public branch.